packages/salesforcedx-apex-replay-debugger/src/core/heapDumpService.ts

```diff
@@ -146,10 +146,7 @@ export class HeapDumpService {
               // the value to a string which seems to work just fine for everything
               // except boolean values. If the boolean values aren't toString'd then
               // they can cause issues in the variable's window.
```

Comment:
Can you elaborate on these issues?

Reply:
I removed the comment but I can comment on the issues. This was the 'weirdness' I'd seen in variables where sometimes things didn't show up correctly. The variable.value is a string and for apparently typescript allows assigning true/false to that string which resulted in the variable sometimes showing up, sometimes not.